named-contributing-orgs-opt-in.yml

```diff
@@ -5,3 +5,8 @@
 named-sponsors:
   - Red Hat
   - Microsoft
+
+# Any organisation that opts in as a sponsor will also be shown on charts of contributors
+# To be shown in the charts, but *not* ever listed as a sponsor, an organisation should add its name to this list
+named-contributing-organisations:
```

Comment:
Organisation or Organization? No preference, but I think using the American English version is more common?

Reply:
Oh gosh. Maybe it should be org to just avoid the whole issue ... ?

Reply:
Good idea, and shorter too 😉